git-keeper-client/gkeepclient/gkeep.py

```diff
@@ -230,6 +231,20 @@ def add_trigger_subparser(subparsers):
                                 'students')
 
 
+def add_config_subparser(subparsers):
+    """
+    Add a subparser for action 'config', which asks to create a new
+    configuration file
+    or to overwrite the existing one
+
+    :param subparsers: subparsers to add to
+    """
+
+    subparser = subparsers.add_parser('config',
+                                      help='create or overwrite configuration'
+                                           'file')
```

Comment:
A space is needed after `configuration` since a space is not added into the string when you split it onto multiple lines in that way.